test/export/FHIRExporter.test.ts

```diff
@@ -0,0 +1,10 @@
+import { exportFHIR } from '../../src/export';
+import { FSHTank } from '../../src/import';
+
+describe('FHIRExporter', () => {
+  it('should output empty results with empty input', () => {
+    const input = new FSHTank([]);
+    const result = exportFHIR(input);
+    expect(result).toEqual([]);
```

Comment:
If you make the changes suggested elsewhere, I guess this should return a `Package` with the `packageJSON` set to the `packageJSON` that was passed in, and with `profiles` and `extensions` both set to empty arrays.